lib/tsconfig.json

```diff
@@ -1,7 +1,8 @@
 {
   "extends": "../tsconfig.json",
   "compilerOptions": {
-    "sourceMap": true,
+    "outDir": "../dist",
```

Comment:
I had problems with IDE in the past separating the source and build directories. Is this required? Source maps are already excluded from the distribution here: https://github.com/Borewit/musicbrainz-api/blob/8a5aaad5cc616ed8f6631827241171c016738e78/.npmignore#L10

Reply:
By specifying an explicit distribution folder, you can more easily control what gets shipped to NPM for download by other projects. Your current configuration outputs built files (.js and .d.ts) alongside your source code (.ts files) and you ship all of it to NPM. By building to the `dist` folder and shipping just that, consumers of the package get just the runnable JS code and optional types.

As for source maps and the dev workflow: the way I usually approach this is by separating my build and dev workflows. I have a set of commands (typically npm scripts) that I use for deving that have source maps and whatever else might make my life easier. And then I have a set of commands for building a distribution files. You can even have a separate tsconfig.json for each if you want to control all aspects of it